### PR TITLE
ACMS-1907: Update deployment scrip for acquia_cms.

### PR DESCRIPTION
**Pull request by chandan-singh7929**
**Motivation**

Fixes #[ACMS-1907]

**Proposed changes**

**Alternatives considered**

**Testing steps**

**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

**Comment by chandan-singh7929** [comment]
Verified changes on ode1 and this seems to be working fine. cc @vishalkhode1

**Comment by chandan-singh7929** [comment]
image
image
image

File: composer.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
         "drupal/acquia_cms_starter": "dev-develop",
         "drupal/acquia_cms_toolbar": "dev-develop",
         "drupal/acquia_cms_tour": "dev-develop",
+        "drupal/consumer_image_styles": "^4.0",
```
Comment (deepakmishra2) [inline composer.json:30]: @chandan-singh7929 why do we need to include this module?

Comment (chandan-singh7929) [reply composer.json:30]: This module is required to setup headless starterkit.

**Review by rajeshreeputra — APPROVED** [review]
Looks good to me.

**Review by deepakmishra2 — APPROVED** [review]

**Review by apathak18 — APPROVED** [review]

**Review by vishalkhode1 — APPROVED** [review]

**Pull request merged**